src/pages/contributors/mark-waite.adoc

```diff
@@ -0,0 +1,85 @@
+His travels also include two years he spent in northern Italy beginning at age 19.
+Mark and his wife still maintain several friendships he first made during those two years in Italy.
+Whether at home or traveling, Mark enjoys recreational cycling and wandering with his wife.
+
+== What is your background prior to contributing to Jenkins?
+
+I graduated from university with a degree in Mechanical Engineering.
+During the interview process, while preparing for my first job after graduating, I told the interviewer that I didn't want to be a mechanical engineer and that I wanted to play with computers for the rest of my life.
+That interviewer was seeking people to create a mechanical engineering software business inside Hewlett-Packard.
+I was one of their support engineers for the first several years.
+
+After spending enough time as a support engineer, I asked if I could become a programmer.
+They were bold enough to allow me to become a programmer.
+However, after some years of programming, I realized I was not a great programmer but a good organizer.
+I decided to switch to release management and then to managing a development team.
+
+In late 2002 or early 2003, my team learned about Extreme Programming, a methodology that encourages all production code to be written by two programmers paired together.
+Extreme Programming requires continuous integration.
+We set our goal to have "XP by 3/03" and started to create our own CI server.
+
+In 2007 or 2008, I switched to a new company with a new project and discovered the continuous integration server that became Jenkins.
+I was impressed at how easy it was to deploy and manage and how well it worked.
+
+A few companies after that, I joined CloudBees to work full-time on helping Jenkins grow.
+
+== How long have you been using Jenkins?
+
+I started using Jenkins in 2007 or 2008.
+I've been using it since then, even during the times when I worked for employers that chose another continuous integration solution.
+
+== Why choose Jenkins over other projects?
+
+Jenkins was easy to configure and easy to manage, especially compared to its competitors at the time.
+I defined jobs, modified configurations, and viewed test results from a web browser.
+I often found that the flexibility of Jenkins was more important to me than the specific reasons that an employer had chosen another solution.
+I had problems to solve and Jenkins solved those problems.
+
+== What problems has Jenkins solved for you?
+
+Jenkins builds my code, runs my tests, and shows the results across a wide range of systems and technologies.
+
+== Is there an aspect of Jenkins that you're particularly passionate about?
+
+I have been involved in each area of Jenkins at some point, but I'm proud of the link:https://plugins.jenkins.io/git/[Git] and link:https://plugins.jenkins.io/git-client/[Git client] plugins that I created and maintain.
```

Comment:
I definitely did not create either of those plugins.  The git plugin was created by Nigel Magnay with significant contributions from Kohsuke Kawaguchi, Nicolas de Loof, Robert Sandell, and many others.  The git client plugin was created by Nicolas de Loof with contributions from Kohsuke Kawaguchi and many others.  I have written many of the tests for those two plugins, but was definitely not the creator of either of them.
```suggestion
I have been involved in each area of Jenkins at some point, but I'm proud of the link:https://plugins.jenkins.io/git/[Git] and link:https://plugins.jenkins.io/git-client/[Git client] plugins that I maintain.
```